.github/ISSUE_TEMPLATE/report-a-question-in-the-wetransform-forum-.md

```diff
@@ -0,0 +1,12 @@
+---
+name: 'Report a question in the wetransform Forum '
+about: On importing or transforming the data into HALE studio or HALE connect
```

Comment:
@emanuelaepure10 The spelling "hale»studio" should always be used, i.e. instead of "HALE studio". The spelling "HALE" was dropped together with the old name "Humboldt Alignment Editor" in version 3.0.0.

Also, IMO the hale repository should not be used for questions regarding hale»connect. In case that reference is kept, the spelling "hale»connect" should be used.

Reply:
@florianesser I have updated the question-issue using just "hale»studio".